play/settings/settings.py

```diff
@@ -99,7 +104,18 @@
     },
 ]
 
+AUTHENTICATION_BACKENDS = (
+    'social_core.backends.github.GithubOAuth2',
+    'social_core.backends.twitter.TwitterOAuth',
```

Comment:
remove twitter and facebook, and maybe the model backend?

Reply:
:+1: good call, will do.